config.sh

```diff
@@ -9,10 +9,16 @@ function pre_build {
     curl -o libev-${LIBEV_VERSION}.tar.gz http://dist.schmorp.de/libev/Attic/libev-${LIBEV_VERSION}.tar.gz
     tar -xzf libev-${LIBEV_VERSION}.tar.gz
     pushd libev-${LIBEV_VERSION}/
-    ./configure
-    make
-    make install
-    popd
+    if (./configure); then
+	echo "Build succeeded, continuing with install"
```

Comment:
This message (and the one below) are fairly obviously incorrect; it's the _configuration_ that fails in this case, not the build. This was added while trying to debug the Python 3.5 + OSX build failure mentioned above, but after that was fixed I decided to keep it around since it could be useful in the future.  It was originally applied to "make" rather than "./configure" by mistake, however, thus the incorrect messaging.